custom/icds_reports/ucr/reports/asr_2_3_person_cases.json

```diff
@@ -8,21 +8,15 @@
     "priyanka-app",
     "shrena-dev",
     "aparatest",
-    "reach-sandbox",
-    "reach-dashboard-qa",
     "reach-test",
-    "icds-test",
-    "icds-sql",
-    "icds-cas",
-    "cas-lab",
-    "icds-cas-sandbox"
+    "icds-cas"
```

Comment:
One thing I was unsure about was whether to use the old or the new list of domains in all of these.

Reply:
I think that we should keep the old ones. I'm pretty sure that the ones deleted here aren't used anymore as more people are using their own sandbox domains, but its probably not worth the effort of verifying that. I wish we did have a verifiable way of 'retiring' a domain from these, but it might take more work on dev and GS.

You may need to add these deleted domains to personcase v3 datasource as well (they shouldn't have data in ICDS, so the extra tables won't add ongoing overhead)

Reply:
updated